client/pages/details.jsx

```diff
@@ -156,4 +193,40 @@ export default class Details extends React.Component {
       })
       .catch(err => console.error(err));
   }
+
+  handleNewReview(event) {
+    event.preventDefault();
+    const { animeInfo } = this.state;
+    const { reviews } = this.state;
+    const updateReviews = reviews;
```

Comment:
Be sure to make a shallow copy of the array before modifying it in any way. You can do this with `Array.prototype.slice()`

```js
const updateReview = reviews.slice();
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice